api/src/pcapi/core/external_bookings/api.py

```diff
@@ -52,6 +61,18 @@ def book_cinema_ticket(
     return client.book_ticket(show_id, booking, beneficiary)
 
 
+def disable_external_bookings() -> None:
+    for feature_name in EXTERNAL_BOOKINGS_FF:
```

Comment:
Ca n'a quasiment pas d'impact ici mais ce serait bien si on faisait des bulk insert/update tout le temps quand on a une boucle for, je pense que ça éviterait d'avoir des dev qui copient colent causant des n+1 (:hand: coupable)

Reply:
J'ai l'impression que ça se fait automatiquement lorsqu'on pas d'autoflush https://docs.sqlalchemy.org/en/20/_modules/examples/performance/bulk_updates.html

Reply:
A priori, si les FF définis dans `EXTERNAL_BOOKINGS_FF` sont tous des éléments de l'enum `FeatureToggle`, alors ils doivent exister en db, puisqu'ils sont automatiquement créés à l'initialisation du code s'ils manquaient. Donc on ne devrait jamais tomber dans le cas d'erreur. On pourrait donc tout faire en une seule requête :
```
feature.Feature.query.filter(feature.Feature.name.in_(EXTERNAL_BOOKINGS_FF)).update({"isActive": True}, synchronize_session=False)
```
au lieu s'un select par FF.
Cela dit, c'est exécuté une seule fois par semaine et uniquement sur staging, donc on s'en moque probablement un peu de l'optimisation dans ce cas précis :-)

Reply:
J'avais fait au plus naïf mais autant faire cela bien; merci pour vos retours.